README.md

```diff
@@ -65,32 +65,21 @@ Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapids
 
 cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
-For `cudf version == 22.06` :
 ```bash
-# for CUDA 11.0
-conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=22.06 python=3.9 cudatoolkit=11.0
-
-# or, for CUDA 11.2
-conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=22.06 python=3.9 cudatoolkit=11.2
-
+# for CUDA 11.5
+conda install -c rapidsai -c conda-forge -c nvidia \
+    cudf cudatoolkit=11.5
+# for CUDA 11.2
+conda install -c rapidsai -c conda-forge -c nvidia \
+    cudf cudatoolkit=11.2
 ```
 
-For the nightly version of `cudf` :
-```bash
-# for CUDA 11.0
-conda install -c rapidsai-nightly -c nvidia -c numba -c conda-forge \
-    cudf python=3.9 cudatoolkit=11.0
-
-# or, for CUDA 11.2
-conda install -c rapidsai-nightly -c nvidia -c numba -c conda-forge \
-    cudf python=3.9 cudatoolkit=11.2
-```
+We also provide [nightly Conda packages](https://anaconda.org/rapidsai-nightly) built from the HEAD
```

Comment:
The edits here are copied from RMM, and should help to make our README more "evergreen."